### ISSUE TITLE
Get-AzAutomationDscNodeConfiguration RollupStatus documentation

### ISSUE THREAD

**Issue by doherty100** [Azure/azure-powershell#17798]
### Description

I have been unable to find any documentation describing what the `RollupStatus` parameter for [Get-AzAutomationDscNodeConfiguration](https://docs.microsoft.com/en-us/powershell/module/az.automation/get-azautomationdscnodeconfiguration?view=azps-7.4.0) is in [Az.Automation](https://docs.microsoft.com/en-us/powershell/module/az.automation/?view=azps-7.4.0). The values `Good` and `Bad` are documented but provided without any context. What does `Good` mean? What does `Bad` mean?

**Comment by jaspkaur28** [comment]
@doherty100 
"Good" means that for this particular node configuration, if all the nodes are in status "Compliant".

"Bad" means that for this particular node configuration, if any one of the nodes is in status which is not "Compliant", like "Failed", "Non Compliant", "Unresponsive".
It is defined in the open-source cmdlets:
 https://github.com/Azure/azure-powershell/blob/4668bff1cc4ed6510d9b0d4c6ec289ab27f04241/src/Automation/Automation/Common/AutomationPSClientDSC.cs 


private string GetRollupStatus(string resourceGroupName, string automationAccountName, string nodeConfigurationName)
{
    var nextLink = string.Empty;
    List<Model.DscNode> nodes = new List<Model.DscNode>();
    do
    {
        nodes.AddRange(this.ListDscNodesByNodeConfiguration(resourceGroupName, automationAccountName, nodeConfigurationName, null, ref nextLink));
    } while (!string.IsNullOrEmpty(nextLink));
    
    foreach (var node in nodes)
    {
        if (node.Status.Equals("Not Compliant") || node.Status.Equals("Failed") || node.Status.Equals("Unresponsive"))
        {
            return "Bad";
        }
    }
    return "Good";
}

**Comment by doherty100** [comment]
> @doherty100 "Good" means that for this particular node configuration, if all the nodes are in status "Compliant". "Bad" means that for this particular node configuration, if any one of the nodes is in status which is not "Compliant", like "Failed", "Non Compliant", "Unresponsive".
> 
> It is defined in the open-source cmdlets: https://github.com/Azure/azure-powershell/blob/4668bff1cc4ed6510d9b0d4c6ec289ab27f04241/src/Automation/Automation/Common/AutomationPSClientDSC.cs
> 
> private string GetRollupStatus(string resourceGroupName, string automationAccountName, string nodeConfigurationName) { var nextLink = string.Empty; List<Model.DscNode> nodes = new List<Model.DscNode>(); do { nodes.AddRange(this.ListDscNodesByNodeConfiguration(resourceGroupName, automationAccountName, nodeConfigurationName, null, ref nextLink)); } while (!string.IsNullOrEmpty(nextLink));
> 
> ```
> foreach (var node in nodes)
> {
>     if (node.Status.Equals("Not Compliant") || node.Status.Equals("Failed") || node.Status.Equals("Unresponsive"))
>     {
>         return "Bad";
>     }
> }
> return "Good";
> ```
> 
> }

@jaspkaur28 thank you should this be included in the documentation [here](https://docs.microsoft.com/en-us/powershell/module/az.automation/get-azautomationdscnodeconfiguration?view=azps-8.0.0#parameters)?

**Comment by jaspkaur28** [comment]
@doherty100 yes , the explanation will be added to the documentation as part of June month sprint.

**Comment by SnehaSudhirG** [comment]
Thank you for the feedback. The info is live and here is the live link - https://docs.microsoft.com/en-us/powershell/module/az.automation/get-azautomationdscnodeconfiguration?view=azps-8.0.0

**Comment by SnehaSudhirG** [comment]
#please-close
